LICENSE.rst

```diff
@@ -2,36 +2,31 @@
+is being made available as a public service.  Pursuant to `title 17 United
+States Code Section 105`_, works of NIST_ employees are not subject to
+copyright protection in the United States.  This software may be subject to
+foreign copyright.  Permission in the United States and in foreign
+countries, to the extent that NIST_ may hold copyright, to use, copy,
+modify, create derivative works, and distribute this software and its
+documentation without fee is hereby granted on a non-exclusive basis,
+provided that this notice and disclaimer of warranty appears in all copies.
 
-To the extent that NIST_ may hold copyright in countries other than the
-United States, you are hereby granted the non-exclusive irrevocable
-and unconditional right to print, publish, prepare derivative works and
-distribute this software, in any medium, or authorize others to do so
-on your behalf, on a royalty-free basis throughout the world.
-
-You may improve, modify, and create derivative works of the software or any
-portion of the software, and you may copy and distribute such modifications
-or works. Modified works should carry a notice stating that you changed
-the software and should note the date and nature of any such change. Please
-explicitly acknowledge the National Institute of Standards and Technology
-as the original source, by linking back to `this repository`_ or
-`the website`_.
-
-This software can be redistributed and/or modified freely provided that any
-derivative works bear some notice that they are derived from it, and any
-modified versions bear some notice that they have been modified.
+THE SOFTWARE IS PROVIDED 'AS IS' WITHOUT ANY WARRANTY OF ANY KIND, EITHER
+EXPRESSED, IMPLIED, OR STATUTORY, INCLUDING, BUT NOT LIMITED TO, ANY
+WARRANTY THAT THE SOFTWARE WILL CONFORM TO SPECIFICATIONS, ANY IMPLIED
+WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND
+FREEDOM FROM INFRINGEMENT, AND ANY WARRANTY THAT THE DOCUMENTATION WILL
+CONFORM TO THE SOFTWARE, OR ANY WARRANTY THAT THE SOFTWARE WILL BE ERROR
+FREE. IN NO EVENT SHALL NIST BE LIABLE FOR ANY DAMAGES, INCLUDING, BUT NOT
+LIMITED TO, DIRECT, INDIRECT, SPECIAL OR CONSEQUENTIAL DAMAGES, ARISING OUT
+OF, RESULTING FROM, OR IN ANY WAY CONNECTED WITH THIS SOFTWARE, WHETHER OR
+NOT BASED UPON WARRANTY, CONTRACT, TORT, OR OTHERWISE, WHETHER OR NOT
+INJURY WAS SUSTAINED BY PERSONS OR PROPERTY OR OTHERWISE, AND WHETHER OR
+NOT LOSS WAS SUSTAINED FROM, OR AROSE OUT OF THE RESULTS OF, OR USE OF, THE
+SOFTWARE OR SERVICES PROVIDED HEREUNDER.
 
 .. _National Institute of Standards and Technology: http://www.nist.gov/
 .. _NIST: http://www.nist.gov/
-.. _the website: https://www.ctcms.nist.gov/fipy
-.. _this repository: https://github.com/usnistgov/fipy
-.. _title 17 section 105: http://uscode.house.gov/uscode-cgi/fastweb.exe?getdoc+uscview+t17t20+9+0++
+.. _title 17 United States Code Section 105: http://uscode.house.gov/uscode-cgi/fastweb.exe?getdoc+uscview+t17t20+9+0++
```

Comment:
Link is broken. Please substitute
https://www.copyright.gov/title17/92chap1.html#105

Reply:
... or, to stay within uscode.house.gov, please use
http://uscode.house.gov/view.xhtml?req=granuleid:USC-prelim-title17-section105&num=0&edition=prelim

Reply:
... or, use the Government Publishing Office (requires specifying the year)
https://www.gpo.gov/fdsys/pkg/USCODE-2017-title17/html/USCODE-2017-title17-chap1-sec105.htm

Reply:
*sigh* That's not what "tkphd approved these changes 2 days ago" means.